### PR TITLE
Improvement calabash-android gen command

### PR DESCRIPTION
**Pull request by fabianofranca**
Hi, update "calabash-android gen" to work like calabash-ios. Add "calabash-android gen-no-questions".

### CODE REVIEW

**Comment by TobiasRoikjer** [comment]
I prefer the Gemfile to always be created, if it does not exist. The gemfile should be part of the skeleton in the project.

**Comment by fabianofranca** [comment]
Ok, my intention is match the two gems versions. I will remove question on iOS version too.

Em 4 de set de 2016 8:16 AM, Tobias Røikjer <notifications@github.com> escreveu:

I prefer the Gemfile to always be created, if it does not exist. The gemfile should be part of the skeleton in the project.

File: ruby-gem/bin/calabash-android-generate.rb [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
-    puts "Please hit return to confirm that's what you want."
-  end
-  exit 2 unless STDIN.gets.chomp == ''
 
-  FileUtils.cp_r(@source_dir, @features_dir)
+  if questions
+    msg("Question") do
+      puts "I'm about to create a features directory here:"
+      puts ""
+      puts "#{ENV["PWD"]}/features"
```
Comment (TobiasRoikjer) [inline ruby-gem/bin/calabash-android-generate.rb]: I don't know if ENV["PWD"] will always be set. We should use a Ruby method to obtain the current working directory.

File: ruby-gem/bin/calabash-android-generate.rb [hunk 2]
```diff
@@ -1,20 +1,63 @@
...
+      puts ""
+      puts "This directory will contain all of your calabash tests."
+      puts "Shall I proceed? (Y/n)"
```
Comment (TobiasRoikjer) [inline ruby-gem/bin/calabash-android-generate.rb]: I don't really want to change the behaviour of the "normal" generate with questions.

Comment (fabianofranca) [inline ruby-gem/bin/calabash-android-generate.rb]: I don't understand. You want remove all questions? The feature create question already existed.

Comment (TobiasRoikjer) [inline ruby-gem/bin/calabash-android-generate.rb]: The question asked has been changed, as well as the keystrokes needed to proceed. I don't see a reason to change them

Comment (fabianofranca) [inline ruby-gem/bin/calabash-android-generate.rb]: My intention is to turn "gen" command the same iOS and Android.. Today, the libs no match. I getting the same questions from calabash-ios and put in calabash-android.

**Pull request closed without merging**